Dockerfile

```
@@ -23,5 +23,7 @@ RUN apk add dumb-init && \
 COPY --from=build /project/target/*.jar /app/app.jar
 WORKDIR /app
 RUN chown -R javauser:javauser /app
-USER javauser
+EXPOSE 9999
+ENV DEMO_SECRET="supersecret"
+USER root
```

Hadolint (2.12.0)

[warning] Last USER should not be root

(DL3002)